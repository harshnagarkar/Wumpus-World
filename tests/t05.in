abcd-efgh
-abcdefgh
id
15
