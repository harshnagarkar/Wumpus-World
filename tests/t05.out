 a b c
 d - e
 f g h

 a b c
 - d e
 f g h

 - b c
 a d e
 f g h

 b - c
 a d e
 f g h

 b c -
 a d e
 f g h

 b c e
 a d -
 f g h

 b c e
 a - d
 f g h

 b - e
 a c d
 f g h

 - b e
 a c d
 f g h

 a b e
 - c d
 f g h

 a b e
 c - d
 f g h

 a b e
 c d -
 f g h

 a b -
 c d e
 f g h

 a - b
 c d e
 f g h

 - a b
 c d e
 f g h

Total nodes: 16526, max depth is: 14
